image.yaml

```diff
@@ -21,8 +21,8 @@ scripts:
     - package: amq-chown
       exec: install.sh
 sources:
-    - artifact: jboss-a-mq-6.2.1.redhat-159.zip
-      md5: 81b2b70492a9429d9fb9ee3b56bd16d6
-      hint: "https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=46681&product=jboss.amq&version=6.2.1&downloadType=securityPatches"
+    - artifact: jboss-a-mq-6.2.1.redhat-186.zip
+      md5: c8acb8802ddb5bc170fce4c831f56d1b
+      hint: "https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=50521&product=jboss.amq&version=6.2.1&downloadType=securityPatches"
```

Comment:
When you're at it- can you add the "Artifact is available on Customer Portal: " string before the link?